imagestreams/httpd-centos.json

```diff
@@ -21,14 +22,54 @@
         },
         "from": {
           "kind": "ImageStreamTag",
-          "name": "2.4-ubi8"
+          "name": "2.4-ubi9"
         },
         "referencePolicy": {
           "type": "Local"
+        }
+      },
+      {
+        "name": "2.4-fc",
+        "annotations": {
+          "description": "Build and serve static content via Apache HTTP Server (httpd) 2.4 on Fedora. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/httpd-container/blob/master/2.4/README.md.",
+          "iconClass": "icon-apache",
+          "openshift.io/display-name": "Apache HTTP Server 2.4 (Fedora)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "sampleRepo": "https://github.com/sclorg/httpd-ex.git",
+          "supports": "httpd",
+          "tags": "builder,httpd",
+          "version": "2.4"
+        },
+        "from": {
+          "kind": "DockerImage",
+          "name": "quay.io/fedora/httpd-24:latest"
         },
-        "name": "latest"
+        "referencePolicy": {
+          "type": "Local"
+        }
       },
       {
+        "name": "2.4-micro-fc",
```

Comment:
Fedora images aren't versioned by Fedora release, so I added the `fc` suffix here for fedora-continuous

Reply:
@LorbusChris For CentOS Stream we use c9s or c8s. Are you sure we want to add suffix. C9S and C8S image leaves in different namespace.

Reply:
I see, the point. we can not have two images named `2.4-micro`.